pkgs/tools/filesystems/httm/default.nix

```diff
@@ -6,29 +6,41 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "httm";
-  version = "0.23.3";
+  version = "0.31.6";
 
   src = fetchFromGitHub {
     owner = "kimono-koans";
     repo = pname;
     rev = version;
-    hash = "sha256-yia7GEPemFVHzTkhrL7HejQsFO1zwpdUtq4DLdm4s2g=";
+    hash = "sha256-iASl4chx0vcS+SWK65URWqsGmucFUavKrDBnnaffLu8=";
   };
 
+  cargoHash = "";
```

Comment:
This shouldn’t be needed.

Reply:
I forgot how to update this. :sweat_smile: 
No matter if with or without that line, I get:
```
 nix profile install -f $NIXPKGS httm
error: build of '/nix/store/s4vqvaw8rsfpim196xg0sg5yxv2qp9lr-httm-0.38.1.drv' on 'ssh://nixBuild@builder' failed: builder for '/nix/store/s4vqvaw8rsfpim196xg0sg5yxv2qp9lr-httm-0.38.1.drv' failed with exit code 1;
       last 10 log lines:
       > ERROR: cargoHash or cargoSha256 is out of date
       >
       > Cargo.lock is not the same in /build/cargo-vendor-dir
       >
       > To fix the issue:
       > 1. Set cargoHash/cargoSha256 to an empty string: `cargoHash = "";`
       > 2. Build the derivation and wait for it to fail with a hash mismatch
       > 3. Copy the "got: sha256-..." value back into the cargoHash field
       >    You should have: cargoHash = "sha256-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX=";
       >
       For full logs, run 'nix-store -l /nix/store/s4vqvaw8rsfpim196xg0sg5yxv2qp9lr-httm-0.38.1.drv'.
error: builder for '/nix/store/s4vqvaw8rsfpim196xg0sg5yxv2qp9lr-httm-0.38.1.drv' failed with exit code 1;
       last 10 log lines:
       > ERROR: cargoHash or cargoSha256 is out of date
       >
       > Cargo.lock is not the same in /build/cargo-vendor-dir
       >
       > To fix the issue:
       > 1. Set cargoHash/cargoSha256 to an empty string: `cargoHash = "";`
       > 2. Build the derivation and wait for it to fail with a hash mismatch
       > 3. Copy the "got: sha256-..." value back into the cargoHash field
       >    You should have: cargoHash = "sha256-XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX=";
       >
       For full logs, run 'nix log /nix/store/s4vqvaw8rsfpim196xg0sg5yxv2qp9lr-httm-0.38.1.drv'.
       ```

Reply:
Removing this line and updating the `Cargo.lock` in nixpkgs worked for me

Reply:
Could you open a new PR then?

Reply:
Here you go: #306783